libpod/oci_conmon_linux.go

```diff
@@ -783,13 +783,29 @@ func (r *ConmonOCIRuntime) ExecContainer(c *Container, sessionID string, options
 	return pid, attachChan, err
 }
 
+func isRetryable(err error) bool {
```

Comment:
A comment would be great to recall in which cases we're returning true and why.

Reply:
added and pushed a new version